src/App.css

```diff
@@ -3,49 +3,74 @@
   justify-content: center;
   align-items: center;
   height: fit-content;
-  margin-top: 60px;
+  margin-top: 28px;
 }
 
 .board-game{
   display: flex;
   justify-content: center;
   flex-wrap: wrap;
-  width: 39%;
+  width: 500px;
 }
 
-.board-header{
+.hard-board {
+  width: 800px;
+}
+
+.board-header, .levels{
   display: flex;
   justify-content: space-between;
-  padding: 0 20px;
+  padding: 0 4%;
 }
 
 .board-heading{
-  font-size: 50px;
-  margin: 20px 0;
+  font-size: 40px;
+  margin: 5px 0;
       
 }
 
 .score {
-  font-size: 30px;
+  font-size: 20px;
+}
+
+.levels{
+  margin: 20px 0;
+}
+
+form{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+form input[type="number"]{
+   margin-right: 12px;
+    width: 100px;
+    height: 25px;
+}
+
+form label{
+  margin-right: 12px;
+  font-size: 18px;
 }
 
 .btn {
   display: inline-block;
   padding: 0.50em 1.5em;
   border: 0.1em solid #FFFFFF;
   margin: auto 1px;
-  border-radius: 0.12em;
+  border-radius: 0.5em;
```

Comment:
Any reason to use `em`

Reply:
Not any specific need, just copied the style from the internet so I can give more focus on the Logic part.